packages/renderer/src/lib/ui/NavRegistryEntry.svelte

```diff
@@ -23,5 +29,10 @@ let { entry, meta = $bindable() }: { entry: NavigationRegistryEntry; meta: Tinro
     {:else if entry.icon.iconImage && typeof entry.icon.iconImage === 'string'}
       <img src={entry.icon.iconImage} width="22" height="22" alt={entry.name} />
     {/if}
+    {#if iconWithTitle && entry.icon}
+      <div class="text-xs text-center max-w-[60px] ml-[2px]" title="Icon title">
```

Comment:
question: what means 'Icon title' ? I mean either we need to be specific with the entry.name or no title